deploy/vagrant/conf/gce.yml.template

```diff
@@ -0,0 +1,69 @@
+# Setting up your Google Cloud Project
+#
+# The Project ID for your Google Cloud Platform account.
+Google_Cloud_Project_ID:
+
+# The Client Email address for your Service Account.
+Service_Acount_Email_Address:
+
+# The location of the JSON private key file matching your Service Account.
+json_Key_Location: 
+
+
+#SSH - Vagrant/Ansible will SSH into VMs for environment setup.
+#
+# This is the path to SSH keys. The easiest way to setup GCE keys is using the 
+# CLI (gcloud compute config-ssh) Which creates metatdata that is provided to 
+# every VM launched. Alternatively, it is possible to add to the GCE metatdata service
+# directly or manually manage keys.
+Key_Path: "~/.ssh/google_compute_engine"
```

Comment:
Thanks. Leaving it without default value seems okay, since jason_Key_Location is empty by default as well.